package.json

```diff
@@ -11,7 +11,8 @@
     "install": "lerna bootstrap",
     "contracts": "yarn workspace @unirep-app/contracts run",
     "frontend": "yarn workspace @unirep-app/frontend run",
-    "relay": "yarn workspace @unirep-app/relay run"
+    "relay": "yarn workspace @unirep-app/relay run",
+    "start": "yarn relay keys & yarn contracts hardhat node & (sleep 6 && yarn contracts deploy && (yarn relay start & yarn frontend start))"
```

Comment:
I'm very nervous to give this command to users. It seems very easy to end up with hanging processes. I'd much rather see a solution that uses nodejs child_process to spawn these processes, and cleanup when the main nodejs process exits.

Reply:
yeah it will be better
I will change it

Reply:
[e1f21c8](https://github.com/Unirep/create-unirep-app/pull/20/commits/e1f21c8ace24b15ca4036fe3d3c80563d883760a)